src/main/kotlin/org/rust/lang/core/psi/ext/RsTraitOrImpl.kt

```diff
@@ -20,3 +20,6 @@ interface RsTraitOrImpl : RsItemElement, RsGenericDeclaration, RsAttrProcMacroOw
 
 val RsTraitOrImpl.expandedMembers: List<RsAbstractable>
     get() = members?.expandedMembers.orEmpty()
+
+val RsTraitOrImpl.explicitMembers: List<RsAbstractable>
```

Comment:
I think now we should carefully check all existing usages of `RsTraitOrImpl.members` and replace some of them either with `RsTraitOrImpl.expandedMembers` (if the macro-generated ones are suitable there) or `RsTraitOrImpl.explicitMembers` (otherwise).

For example, we could review the usages from the following classes:
* `org.rust.ide.refactoring.extractTrait.RsExtractTraitHandler`
* `org.rust.ide.refactoring.generate.constructor.GenerateConstructorHandler`

Secondly, I am not sure if we really need `RsMembers.expandedMembers` since we already have `RsTraitOrImpl.expandedMembers`. So maybe we could delete this properly to avoid confusion in the future.

@vlad20012 Maybe you also have some thoughts?

Reply:
Another option is removing `RsTraitOrImpl.expandedMembers` and using `RsMembers.expandedMembers` or newly introduced `RsMembers.explicitMembers` =D

I'm not sure which is better, but I also find `members`/`expandedMembers`/`explicitMembers` confusing. We can also rename `getMembers()` property/function in the `RsTraitOrImpl` and in the grammar as well (and choose a better name to highlight its low-level nature, like `rawMembers`).

Reply:
But, to be honest, we currently have no naming style that highlight differences between raw and macro-expanded and cfg-checked parts of PSI

Reply:
> But, to be honest, we currently have no naming style that highlights differences between raw and macro-expanded and cfg-checked parts of PSI

Right, I think it would be great to introduce the different naming for them to avoid confusion